sdk/src/main/java/com/mapbox/maps/renderer/RenderHandlerThread.kt

```diff
@@ -37,7 +37,9 @@ internal class RenderHandlerThread {
   }
 
   fun stop() {
-    handlerThread.quit()
+    // quit and wait until all the messages will be processed - DESTROY_RENDERER messages
+    // are still there
+    handlerThread.quitSafely()
```

Comment:
As discussed - may affect performance as we'll block main Android UI thread. Assuming destroy render tasks are not related to GPU resources and that we have additional mutex control to block UI thread until EGL is cleared - perhaps remove join at all then?

Reply:
Well `quitSafely` itself will wait for all the tasks to be executed - `join` won't add anything in that case.

Reply:
At the same time join seems indeed unneeded, lets remove it then.